src/Workspaces/Core/Portable/Shared/Extensions/SymbolDisplayPartKindExtensions.cs

```diff
@@ -1,82 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using Microsoft.CodeAnalysis.Compilers;
-using Microsoft.CodeAnalysis.Services.Classification;
-using Microsoft.CodeAnalysis.Text;
-using Microsoft.CodeAnalysis.Utilities;
-
-namespace Microsoft.CodeAnalysis.Services.Shared.Extensions
-{
-    internal static class SymbolDisplayPartKindExtensions
```

Comment:
this was not part of solution, file in github but not referenced in any where in solution. seems file left out while other changes.